### PR TITLE
Audit dashboard (migration from dashboards-library)

### PR DESCRIPTION
**Pull request by kidk**
Continuing the migration of quickstart dashboards this adds all the `audit` dashboards I could find and adds them in a centralised directory.

### CODE REVIEW

File: quickstarts/audit/audit-events-analysis/config.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+id: 91b45072-df61-4854-8d3f-76766e75d2a4
```
Comment (jpvajda) [inline quickstarts/audit/audit-events-analysis/config.yaml:1]: curious why there is an ID here... did this get added by copy paste?

File: quickstarts/audit/audit-events-analysis/config.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - audit events
+  - new relic
+
+documentation:
+  - name: Query account audit logs
+    url: https://docs.newrelic.com/docs/data-apis/understand-data/event-data/query-account-audit-logs-nrauditevent/
+    description: Additional information on the New Relic Audit events
```
Comment (jpvajda) [inline quickstarts/audit/audit-events-analysis/config.yaml:43]:
```suggestion
    description: | Additional information on the New Relic Audit events
```

File: quickstarts/audit/audit-events-analysis/config.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  - Swati Joshi
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - audit events
+  - new relic
+
+documentation:
+  - name: Query account audit logs
+    url: https://docs.newrelic.com/docs/data-apis/understand-data/event-data/query-account-audit-logs-nrauditevent/
```
Comment (jpvajda) [inline quickstarts/audit/audit-events-analysis/config.yaml:42]:
```suggestion
    url: >-
       https://docs.newrelic.com/docs/data-apis/understand-data/event-data/query-account-audit-logs-nrauditevent/
```

File: quickstarts/audit/k8s-data-ingestion-analysis/config.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
+id: 15c23bf4-3dfa-47ba-850d-362bd8600fa0
```
Comment (jpvajda) [inline quickstarts/audit/k8s-data-ingestion-analysis/config.yaml:1]: Same question as above on the use of an ID.

File: quickstarts/audit/k8s-data-ingestion-analysis/config.yaml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+keywords:
+  - k8s
+  - kubernetes
+  - data ingest
+  - audit
+
+documentation:
+  - name: New Relic Kubernetes integration
+    url: https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration/
+    description: Introduction to the New Relic Kubernetes integration
```
Comment (jpvajda) [inline quickstarts/audit/k8s-data-ingestion-analysis/config.yaml:34]:
```suggestion
    description: | Introduction to the New Relic Kubernetes integration
```

File: quickstarts/audit/k8s-data-ingestion-analysis/config.yaml [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - k8s
+  - kubernetes
+  - data ingest
+  - audit
+
+documentation:
+  - name: New Relic Kubernetes integration
+    url: https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration/
```
Comment (jpvajda) [inline quickstarts/audit/k8s-data-ingestion-analysis/config.yaml:33]:
```suggestion
    url: >- 
      https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration/
```

File: quickstarts/audit/k8s-log-ingestion-analysis/config.yaml [hunk 7]
```diff
@@ -0,0 +1,36 @@
+id: f8bb7301-ac34-4b3c-9613-422266beb0e0
```
Comment (jpvajda) [inline quickstarts/audit/k8s-log-ingestion-analysis/config.yaml:1]: Same ID question as above

File: quickstarts/audit/k8s-log-ingestion-analysis/config.yaml [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+  - k8s
+  - kubernetes
+  - data ingest
+  - audit
+  - logs
+
+documentation:
+  - name: New Relic Kubernetes Logs integration
+    url: https://docs.newrelic.com/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/
+    description: Kubernetes plugin for log forwarding
```
Comment (jpvajda) [inline quickstarts/audit/k8s-log-ingestion-analysis/config.yaml:36]:
```suggestion
    description: | Kubernetes plugin for log forwarding
```

File: quickstarts/audit/k8s-log-ingestion-analysis/config.yaml [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+keywords:
+  - k8s
+  - kubernetes
+  - data ingest
+  - audit
+  - logs
+
+documentation:
+  - name: New Relic Kubernetes Logs integration
+    url: https://docs.newrelic.com/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/
```
Comment (jpvajda) [inline quickstarts/audit/k8s-log-ingestion-analysis/config.yaml:35]:
```suggestion
    url: >- 
      https://docs.newrelic.com/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/
```

**Review by jpvajda — CHANGES_REQUESTED** [review]
Thanks for the contribution! I had some suggestions.

**Comment by jpvajda** [comment]
There is also a conflict on this PR that will need resolution.

**Review by jpvajda — DISMISSED** [review]
I had a few suggestions, thanks for the contribution.

File: quickstarts/audit/audit-events-analysis/config.yaml (hunk 1 above)
Comment (kidk) [reply quickstarts/audit/audit-events-analysis/config.yaml:1]: I added those, does this cause problems?

Comment (jpvajda) [reply quickstarts/audit/audit-events-analysis/config.yaml:1]: let me check with @aswanson-nr, I'm not 100% sure how we handle the `IDs` for these config files, maybe they are manually added? 🤔

Comment (rudouglas) [reply quickstarts/audit/audit-events-analysis/config.yaml:1]: Yes @kidk they are auto-generated as part of the Release workflow [here](https://github.com/newrelic/newrelic-quickstarts/blob/e73caa3ca98272eb27a9b25cf7d7b13964120b37/.github/workflows/release.yml#L32) so if you could remove these that would be great

Comment (rudouglas) [reply quickstarts/audit/audit-events-analysis/config.yaml]: @kidk i went ahead and did this for you 🤠

**Review by rudouglas — APPROVED** [review]
looking goood

**Comment by kidk** [comment]
Thank you both 👍

**Pull request merged**